README.MD

```diff
@@ -101,8 +101,8 @@ Feel free to contribute to the project and stay tuned for more exciting updates!
 📝 Getting Started
 To use the app, follow these steps:
 
-1. Create YouTube API V3 credentials.
-2. Add the credentials to `local.properties` like  `API_KEY =  Your_API_VALUE`.
+1. Create [YouTube API V3 credentials](https://developers.google.com/youtube/v3/getting-started).
+2. Add the credentials to `local.properties` like  `API_KEY = "Your_API_KEY_VALUE"`.
```

Comment:
I received a compilation error without the quotes

Reply:
Yeah, that's due to new versions of Napier but everything works fine.

Reply:
thats the code that is generated without quotes:
![image](https://github.com/KhubaibKhan4/Youtube-Clone-KMP/assets/36509607/9ae9e715-d8cc-4eb8-b834-c0d21101b376)

Reply:
You need to add the API key inside the local.properties file, then this error will be resolved.

Reply:
alternatively you can bump BuildConfig:
https://github.com/KhubaibKhan4/Youtube-Clone-KMP/blob/99cbff820056d096ef7339d31d1daf78c6593f91/gradle/libs.versions.toml#L22
buildConfig = "5.3.5"

and change 
https://github.com/KhubaibKhan4/Youtube-Clone-KMP/blob/0fbd8d7cf48e731189fd6e23b628e19697452ecf/composeApp/build.gradle.kts#L206
to 
buildConfigField("API_KEY",properties.getProperty("API_KEY"))

then it is able to put those quotes in for you

Reply:
But it's already working fine mate. If you are having issues, we can deal with them on Slack.

Reply:
I did, but it needs quotes is what I am saying:
![image](https://github.com/KhubaibKhan4/Youtube-Clone-KMP/assets/36509607/b1ca6e03-5077-4087-84c1-b13adb0d46c2)

just wanted to clarify this in the readme so future users will not be confused

but all good, I think this is not going anywhere 😄